DynamicPackagingVODContent/StreamHLSAndDASH/src/main/java/sample/StreamHLSAndDASH.java

```diff
@@ -182,7 +201,8 @@ private static void runStreamHlsAndDash(ConfigWrapper config) {
             if (scanner != null) {
                 scanner.close();
```

Comment:
Again, what is this Scanner thing?  Why are we closing it?

Reply:
its the Java console readline equivalent.